.github/workflows/docker-build-static.yml

```diff
@@ -357,25 +340,25 @@ jobs:
 ### Download artifacts, clean up ui, upload to release page ###
 ###############################################################
   create-release-artifacts:
-     needs: [build-ui,build-armhf,build-aarch64,build-amd64]
+     needs: [lldap-test]
```

Comment:
Why `lldap-test`? I'd rather just depend on the builds.

Reply:
Im thinking that that it should pass test first?

Reply:
oh, right, for release it makes sense. Although it's already checked when submitting the PR.

Reply:
gonna revert to old behavior then